### DISCUSSION TITLE
Container image mcr.microsoft.com/dotnet/aspire-dashboard for v8.1.0

### DISCUSSION THREAD

**Discussion by devurandom** [microsoft/aspire#5077] [Q&A] [answered]
Version 8.1.0 of Aspire was released 2 days ago (cf. https://github.com/dotnet/aspire/releases/tag/v8.1.0 and https://devblogs.microsoft.com/dotnet/whats-new-in-aspire-8-1/), but there is no `8.1.0` tag for the `mcr.microsoft.com/dotnet/aspire-dashboard` container image yet (see https://mcr.microsoft.com/en-us/product/dotnet/aspire-dashboard/tags).  Will that come later?  If so, when can I expect it?

**Comment by davidfowl** [comment]
cc @joperezr

**Reply by joperezr** [reply, marked as answer]
Thanks for starting the discussion @devurandom. We created the PR a few days ago to get the new image updated but it has taken a bit longer than expected to get it in. That said, we expect it to get merged today so it should get published really soon: https://github.com/dotnet/dotnet-docker/pull/5720

**Reply by devurandom** [reply]
See-also: https://github.com/dotnet/dotnet-docker/pull/5732
